docs/quickstarts/packet.md

```diff
@@ -0,0 +1,281 @@
+# Lokomotive Packet installation guide
+
+## Contents
+
+* [Introduction](#introduction)
+* [Requirements](#requirements)
+* [Step 1: Install lokoctl](#step-1-install-lokoctl)
+* [Step 2: Set up a temporary working directory](#step-2-set-up-a-temporary-working-directory)
+* [Step 3: Set up credentials from environment variables](#step-3-set-up-credentials-from-environment-variables)
+* [Step 4: Define cluster configuration](#step-4-define-cluster-configuration)
+* [Step 5: Create Lokomotive cluster](#step-5-create-lokomotive-cluster)
+* [Verification](#verification)
+* [Cleanup](#cleanup)
+* [Troubleshooting](#troubleshooting)
+* [Conclusion](#conclusion)
+* [Next steps](#next-steps)
+
+## Introduction
+
+This quickstart guide walks through the steps needed to create a Lokomotive cluster on Packet with
+Flatcar Container Linux using Route53 as the DNS provider.
+
+By the end of this guide, you'll have a working Kubernetes cluster with 3 worker nodes and 3
```

Comment:
Should we avoid mentioning specific settings here, which could be changed by the user?  
```suggestion
By the end of this guide, you'll have a  production-ready Kubernetes cluster running on Packet.
```